validator/client/src/main/java/tech/pegasys/teku/validator/client/AttestationDutyLoader.java

```diff
@@ -74,7 +77,11 @@ public AttestationDutyLoader(
   private SafeFuture<Void> scheduleDuties(
       final ScheduledDuties scheduledDuties, final AttesterDuty duty) {
     final Validator validator = validators.get(duty.getPublicKey());
-    final int aggregatorModulo = CommitteeUtil.getAggregatorModulo(duty.getCommitteeLength());
+    final int aggregatorModulo =
+        specProvider
+            .atSlot(duty.getSlot())
```

Comment:
Note this is using the attestation slot which is different to what was done in the validation code.  I think this one is the way to go.

Reply:
this was copying the original call which pointed at duty. i'll have to check the other one to see why i might have done something different...